packages/utilities/gel-foundations/README.md

```diff
@@ -31,17 +31,19 @@ import { css } from 'styled-components';
 
 const SomeStyledComponent = css`
   font-family: ${GEL_FF_REITH_SANS};
+  ${getLongPrimer(script)};
```

Comment:
Can you please add an example of where 'script' comes from

Reply:
@dr3 Updated. But I would prefer to wait until @sareh approves the rest of the changes in this PR, before updated it again, because it might change.